packages/vite-plugin/src/main.ts

```diff
@@ -111,7 +111,8 @@ export default ({
   const projectLookup: Record<string, ProjectData> = {};
   for (const url of typeof project === 'string' ? [project] : project) {
     const {name} = path.parse(url);
-    const data = {name, url};
+    const metaData = getMeta(`./src/${name}.meta`);
```

Comment:
Projects can be located anywhere in the `src` directory.
Take a look at how the project loader creates the meta path:
https://github.com/motion-canvas/motion-canvas/blob/cdd3df1bff25b04b905e289264831d8d328caaab/packages/vite-plugin/src/main.ts#L206-L207

Reply:
Ok, got it!

Reply:
@aarthificial please have a look now, I have used the path library as the load function uses it.